source/includes/driver-compatibility-caveat.rst

```diff
@@ -0,0 +1,6 @@
+.. note::
+
+   The compatibility matrices show which versions of the driver are
+   recommended for use with a given major release. Older driver versions
+   may be able to connect to a version of MongoDB which is not officially
+   supported, but not all features will be available.
```

Comment:
If old versions of pymongo can still connect to MongoDB 3.4 -- well, MongoDB 3.4 is a supported version.   The phrasing seems to suggest that MongoDB does not support MongoDB 3.4.